### DISCUSSION TITLE
Single Color auto selectd

### DISCUSSION THREAD

**Discussion by GreatBrickLab** [rgriebl/brickstore#764] [Ideas]
I have been all about speed to add parts with the new integration with Brickognize I am able to fly logging parts.  

I would like to recommend a change in the way a color is selected.    When adding a part I have been using the double_click to add.  That works only when the color is also selected.  I suggest that the for parts that have only one known color,  that color be auto selected.  Then I could save one motion of the mouse and a click on the color, and just keep swapping out parts under the camera and double click to add.

Thank you for all you do for LEGO and  BrickLink store owners.
Chris
GBL

**Comment by ZZJHONS** [comment]
+1! please upvote issue #150

**Comment by GreatBrickLab** [comment]
This is in issue #150
